.circleci/config.yml

```diff
@@ -319,6 +347,7 @@ workflows:
           desired-capacity: 1
           requires:
           - build_and_test
+          - sam_deploy_staging
```

Comment:
I think this is redundant/misleading: 
- This job (`code_deploy_production`) already depends on `sam_deploy_production`
- `sam_deploy_production` depends on `code_deploy_staging`
- `code_deploy_staging` depends on `sam_deploy_staging`
- So this job (`code_deploy_production`) is already indirectly dependent on `sam_deploy_staging`.

I realise this is true of a lot of the `build_and_test` entries, but I can imagine situations where you want to go straight to these jobs with only running build and test. Less so with this one.